specs/scenarios/trust-store-and-trust-policy.md

```diff
@@ -0,0 +1,212 @@
+
+```
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-kubernetes",
+            "registryScopes": [ "k8s.wabbitnetworkscr.io/k8s" ],          
+            "signatureVerification": {          
+              "level" : "strict" 
+            },
+            "trustStores": ["ca:wabbit-networks"], 
+            "trustedIdentities": [
+              "x509.subject: C=US, ST=WA, L=Seattle, O=wabbitnetworks.com, OU=KubernetesTeam, CN=KuberenetesTeamSecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+The above *trust store* and *trust policy* are the initial state for each Kubernetes node in the cluster once the node is spun up. Once again, those are baked into the VM image that is used to start the node.
+
+### Adding Trust Certificates to the Trust Store and Trust Policy for the Service Layer
+Once the node is started the Wabbit Networks Batch service needs to be deployed on the node. For that to work, the *trust store* needs to be updated with the Wabbit Networks Batch certificates and a trust policy needs to be added. Here is how the *trust store* should look like:
+
+```
+$XDG_CONFIG_HOME/notation/trust-store
+    /x509
+        /ca
+            /wabbit-networks
+                wabbit-networks-kubernetes.crt
+                wabbit-networks-batch.crt
+        /tsa
+            /publicly-trusted-tsa
+                tsa-cert1.pem
+```
+And here is how the policy should look like before deployint the Wabbit Networks Batch service:
+
+```
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-kubernetes",
+            "registryScopes": [ "k8s.wabbitnetworkscr.io/k8s" ],          
+            "signatureVerification": {          
+              "level" : "strict" 
+            },
+            "trustStores": ["ca:wabbit-networks"], 
+            "trustedIdentities": [
+              "x509.subject: C=US, ST=WA, L=Seattle, O=wabbitnetworks.com, OU=KubernetesTeam, CN=KuberenetesTeamSecureBuilder"
+            ]
+        },
+        {
+            "name": "wabbit-networks-batch",
+            "registryScopes": [ "batch.wabbitnetworkscr.io/batch" ],          
+            "signatureVerification": {          
+              "level" : "strict" 
+            },
+            "trustStores": ["ca:wabbit-networks"], 
+            "trustedIdentities": [
+              "x509.subject: C=US, ST=WA, L=Seattle, O=wabbitnetworks.com, OU=BatchTeam, CN=BatchTeamSecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+### Adding Trust Certificates to the Trust Store and Trust Policy for the Application Layer
+Once the node is started and the Wabbit Networks Batch service is deployed on the node, a customer can deploy their workloads. For that to work, the *trust store* needs to be updated with the Wabbit Networks Batch certificates and a trust policy needs to be added. Here is how the *trust store* should look like:
```

Comment:
same question as previous: Who will add the trust store and trust policy?

Reply:
Same answer as above :) This should be enabled for any tool to do those changes.